src/bidiMapper/domains/context/browsingContextImpl.ts

```diff
@@ -24,6 +24,13 @@ import {LogManager} from '../log/logManager.js';
 import {Protocol} from 'devtools-protocol';
 import {Realm} from '../script/realm.js';
 import {RealmStorage} from '../script/realmStorage.js';
+// import debug from 'debug';
```

Comment:
I cannot use `debug` here directly because it is in the bidiMapper. It would require some polyfills (=extra dependencies).

Should I go ahead with the polyfills, or try something else here?

Reply:
Let's use the logger function from another PR. We do not want indeed to use debug here

Reply:
I cannot do `import {log} from '../../../bidiTab/mapperTabPage.js';`, or else:

```
src/bidiMapper/domains/context/browsingContextImpl.ts:27:19 - error TS6059: File '/Users/tperrotta/Projects/chromium-bidi/src/bidiTab/mapperTabPage.ts' is not under 'rootDir' '/Users/tperrotta/Projects/chromium-bidi/src/bidiMapper'. 'rootDir' is expected to contain all source files.

27 import {log} from '../../../bidiTab/mapperTabPage.js';
                     ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

What did you mean here?

Reply:
Offline: Inject the logger

Reply:
Done